influxdata/tokenizer_test.go

```diff
@@ -0,0 +1,106 @@
+package influxdata
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"testing/iotest"
+
+	qt "github.com/frankban/quicktest"
```

Comment:
My suggestion would be to use the same testing libraries that we use in other places. This one does seem slick.

Reply:
We should have a conversation about this next week.